yrs/src/event.rs

```diff
@@ -0,0 +1,89 @@
+use rand::RngCore;
```

Comment:
Sweet. If you plan to integrate this, could we maybe do that as an outside layer outside of `yrs`? The language bindings certainly don't need that and should just observe changes based on the dropped transactions. On the other hand, we need an API for Rust code.

Maybe it would make sense to separate the efforts a little bit more. We could have..

* `y-base-crdt` as the basis implementation of the CRDT (what `yrs` is right now)
* `y-binding-wrapper` as the wrapper around `y-base-crdt` that can be easily exported to other languages. This doesn't expose a lot of internal information.
* `yrs` The Rust port of Yjs, which simply includes `y-base-crdt` and provides a nice idiomatic Rust API.
* `ypy` Python port
* `ygo` Go port
* ...

WDYT?

Reply:
I'm still thinking about the best way of project organization. So far I think that we could have `y-rs` crate as a core implementation that might be also Rust-idiomatic. I'm curious to try if C FFI bindings (or even WASM bindings) could be just realized as [feature flags](https://doc.rust-lang.org/cargo/reference/features.html) to that crate.

But for now I wouldn't put that into highlight. Originally we planed to make crates in 3rd milestone. Reorganizing code around is cheap when done separately and not mixed with adding other features (like in this branch xD).

We should keep in mind that bindings cannot have generic and lifetime parameters in extern languages APIs and to make sure that dependencies we use can work with WASM/C (eg. `rand` crate worked with WASM but only to version 0.6.3, later ones didn't).